_data/reference.yml

```diff
@@ -66,13 +75,20 @@ toc:
     - docs/user-guide/kubectl/kubectl_config_view.md
     - docs/user-guide/kubectl/kubectl_convert.md
     - docs/user-guide/kubectl/kubectl_cordon.md
+    - docs/user-guide/kubectl/kubectl_cp.md
     - docs/user-guide/kubectl/kubectl_create.md
     - docs/user-guide/kubectl/kubectl_create_configmap.md
+    - docs/user-guide/kubectl/kubectl_create_deployment.md
     - docs/user-guide/kubectl/kubectl_create_namespace.md
+    - docs/user-guide/kubectl/kubectl_create_quota.md
     - docs/user-guide/kubectl/kubectl_create_secret_docker-registry.md
     - docs/user-guide/kubectl/kubectl_create_secret.md
     - docs/user-guide/kubectl/kubectl_create_secret_generic.md
+    - docs/user-guide/kubectl/kubectl_create_secret_tls.md
     - docs/user-guide/kubectl/kubectl_create_serviceaccount.md
+    - docs/user-guide/kubectl/kubectl_create_service_clusterip.md
+    - docs/user-guide/kubectl/kubectl_create_service_loadbalancer.md
+    - docs/user-guide/kubectl/kubectl_create_service_nodeport.md
```

Comment:
@devin-donnelly If sorted alphabetically, should these three docs be before or after `kubectl_create_serviceaccount.md`?